### ISSUE TITLE
Query: flashing mid animation

### ISSUE THREAD

**Issue by k8fisher9** [johnpolacek/superscrollorama#172]
Ive been experimenting with a number of jquery animation libraries, Ive been editing the demo file from superscrollorama (mainly because I couldnt get on very well with scrollmagic.

Im trying to execute 3 wipes one after the other (Ive started with the demo file on superscrollorama) and have the code below. The thing I can see looks like a flicker part way through the wipe slide.

I was wondering is there anything which stands out from the code below? Apologies if there is something obvious, Ive compared the problem Im seeing to those reported in the past caused by user error but cant see where the flicker is coming from.

Is it feasible to check over below at all?

Appreciate any thoughts.

K

<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8"> 
	<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"> 
	<title>...</title>
	<link rel="stylesheet" href="css/normalize.css" type="text/css">
	<link rel="stylesheet" href="css/style.css" type="text/css">	
</head>
<body>

	<script>
	$(document).ready(function() {
		var controller = $.superscrollorama({
			triggerAtCenter: false,
			playoutAnimations: true
		});
	});

	$(window).resize(function () {
		controller.triggerCheckAnim();
	});
	</script>

<div style="background:#ffffff;position:fixed;color:#000000;font-family:'Raleway';font-size:30pt;z-index:9999999;top:0;left:0;width:100%;height:130px"> 
This is the CLS menu
<br>
</div>
	<div id="content-wrapper">
		<div id="First" ><h1>CLS provide innovative search and legal indemnity solutions to aid property transactions throughout the UK and Europe</h1></div>

		<div id="examples-pin">
			<div id="pin-frame-slide" class="pin-frame"><h2>Buying and Selling</h2></div>
			<div id="pin-frame-wipe" class="pin-frame"><h2>Property Development</h2></div>
		</div>
	</div>
	<script type="text/javascript" src="js/greensock/TweenMax.min.js"></script>
	<script src="//ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.min.js"></script>
  	<script>window.jQuery || document.write('<script src="js/jquery-1.9.1.min.js"><\/script>')</script>
  	<script src="js/jquery.lettering-0.6.1.min.js"></script>
	<script src="js/jquery.superscrollorama.js"></script>
	<script>
		$(document).ready(function() {
			$('body').css('visibility','visible');
		
			// TimelineLite for title animation, then start up superscrollorama when complete
			(new TimelineLite({onComplete:initScrollAnimations}))
				
				.from("#newversion", .4, {scale: 5, autoAlpha: 0, ease: Elastic.easeOut})
				.to( $('#title-info'), .5, {css:{opacity:.99, 'margin-top':0}, delay:-1, ease:Quad.easeOut});
			
			function initScrollAnimations() {
				var controller = $.superscrollorama();	
				
				// set duration, in pixels scrolled, for pinned element
				var pinDur = 6000;
				// create animation timeline for pinned element
				var pinAnimations = new TimelineLite();
				pinAnimations
					.append(TweenMax.from($('#pin-frame-pin h2'), .5, {css:{marginTop:0}, ease: Quad.easeInOut}))
					.append([
						TweenMax.to($('#pin-frame-slide'), 1, {css:{marginLeft:0}}),
						TweenMax.to($('#pin-frame-pin'), 1, {css:{marginLeft:'100%'}})
					], .5)
					.append([
						TweenMax.to($('#pin-frame-wipe'), .5, {css:{top:0}}),
						TweenMax.from($('#pin-frame-wipe h2'), .5, {css:{marginTop:'-800px'}})
					], .5)

				// pin element, use onPin and onUnpin to adjust the height of the element
				controller.pin($('#examples-pin'), pinDur, {
					anim:pinAnimations, 
					onPin: function() {
						$('#examples-pin').css('height','100%');
					}, 
					onUnpin: function() {
						$('#examples-pin').css('height','800px');
					}
				});

			}
		});
	</script>
</body></html>

CSS (pretty much untouched from demo)
/*
	COLORS
 
	Dark Blue 		#222438
	Medium Blue		#69697A
	Light Blue	 	#C5C3DE
	Yellow			#FFB000
	Red				#D92B00
	
*/

@font-face {
    font-family: 'ralewaybold';
    src: url('../fonts/raleway-SemiBold.ttf') format('truetype');
    font-weight: bold;
    font-style: bold;

}

@font-face {
    font-family: 'ralewaythin';
    src: url('../fonts/raleway_thin-webfont.eot');
    src: url('../fonts/raleway_thin-webfont.eot?#iefix') format('embedded-opentype'),
         url('../fonts/raleway_thin-webfont.woff') format('woff'),
         url('../fonts/raleway_thin-webfont.ttf') format('truetype'),
         url('../fonts/raleway_thin-webfont.svg') format('svg');
    font-weight: normal;
    font-style: normal;

}

@font-face {
    font-family: 'ralewayLight';
    src: url('../fonts/raleway-Light.ttf') format('truetype');
    font-weight: normal;
    font-style: normal;

}

@font-face {
    font-family: 'raleway';
    src: url('../fonts/raleway-Regular.ttf') format('truetype');
    font-weight: normal;
    font-style: normal;

}

body {
	background: url('../images/blue-gradient-1024x768.jpg') no-repeat!important;
	background-size: cover!important;
	font-weight: bold;
	font-size: 20px;
	text-align: center;
	/*visibility: hidden;*/
}

#CLSMenu{
background:#ffffff;
position:fixed;
color:#000000;
font-family:'Raleway';
font-size:30pt;
z-index:9999999;
top:0;
left:0;
width:100%;
height:130px;
}

#First{
/*background: url('../images/blue-gradient-1024x768.jpg') no-repeat!important;
background-size: cover!important;*/
	font-family: 'raleway',sans-serif!important;
	color:#ffffff!important;
	width:100%!important;
	z-index:0;
	position:fixed;
	top:130px;
	left:0px;
	height:800px!important;
	
}

.simple-demo, .mobile-demo {
	visibility: visible;
}

.simple-demo #content-wrapper {
	height: 1600px;
	padding-top: 400px;
}

h1 {
	margin-top:15%!important;
	font-size: 45px;
	font-weight: normal;
margin: 0 auto;
	font-family: 'raleway',arial,sans-serif;
	color: #ffffff;
	text-align: justify;
    text-justify: inter-word;
	-moz-text-align-last: center;
    text-align-last: center;
	line-height: 45px;
	width:750px!important;
}

h2 {
	font-size: 32pt;
	font-weight: normal;
	margin: 0 auto;
	color: #ffffff;
	margin-top:15%!important;
	font-family: 'raleway',arial,sans-serif;
	color: #ffffff;
	text-align: justify;
    text-justify: inter-word;
	-moz-text-align-last: center;
    text-align-last: center;
	line-height: 45px;
	width:750px!important;

}
h3 {
	font-size: 60px;
	font-weight: normal;
	margin: 0;
	color: #FFB000;
	text-shadow: 0 2px 1px #000;
	font-family: 'raleway',arial,sans-serif;
	text-transform: uppercase;	
}

p {	width: 80%; margin: 20px auto; text-shadow: 0 2px 1px #000; color: #C5C3DE; line-height: 1.4; }
a, a:visited { 
	color: #FFB000; 
	font-weight: bold; 
	font-style: italic;
	text-decoration: none;
	border-bottom: dotted 1px #FFB000;
}
figure > a, #newversion a, .banner {
	border-bottom: none;
}
figcaption { text-shadow: 0 2px 1px #000; color: #C5C3DE; }

ul { list-style: none; }
.download { padding: 10px; }

pre { font-size: 15px; }
code { font-style: normal; }
blockquote {
	text-align: left;
	width: 720px;
	margin: 10px auto;
	background: #C5C3DE;
	border: solid 2px #69697A;
	padding: 0 40px;
}

#console {
	display: none;
	position: fixed;
	top: 0;
	left: 15px;
	width: 500px;
	margin: 0;
	font-size: 14px;
	font-style: italic;
	background-color: #69697A;
	text-shadow: none;
	text-align: left;
	padding: 3px 10px;
	border-radius: 0 0 10px 10px;
	z-index: 99;
	opacity: .8;
}

#title {
	overflow: hidden;
	font-size: 180px;
	margin: 0 auto -580px;
	width: 100%;
	height: 1000px;
	position: relative;
	padding-top: 30px;
	z-index: 99;
}

.title-line { position: relative; display: block; margin: auto; width: 300%; left: -100%; }
#title-line2 { top: -60px; }
#title-line3 { top: -120px; }
.title-line span { display: inline-block; line-height: 1.1; position: relative; }
.twitter-follow-button { position: relative; top: 8px; left: 4px; }
#title-info { position: relative; z-index:99998; margin-top: 40px; opacity: 0; }

.credit {
	font-size: 20px;
	margin: 10px auto;
	text-align: center;
	font-style: italic;
	color: #69697A;
	z-index:100;
}

#credits{
height: 0px !important;
z-index:0!important;
padding:0px!important;
margin:0px!important;
}

#download {
	position: relative;
	margin: 50px auto;
	font-size: 18px;
	width: 100%;
	line-height: 1.8;
	z-index: 99999;
}
#hire { margin-bottom: 80px; }

#download a { color: #69697A; border-bottom: none; text-decoration: underline; }

#examples-1 { overflow: hidden; }
#fly-it { position: relative; }
#scale-it { width: 500%; margin: 0px -200%; padding: 0; }

#examples-pin { position: relative; width: 100%; height: 1200px!important; display:block!important;overflow: hidden; margin-bottom: -200px; }
.pin-frame { position: absolute; width: 100%; height: 100%; overflow: hidden; }
.pin-frame h2 { margin-top: 120px; }
#pin-frame-slide { background: #D92B00; margin-left:-100%; }
#pin-frame-wipe  { background: #FFB000; top:100%; } /*BUYING PALE GREY*/
#pin-frame-wipe h2  { color: #D92B00; }
#pin-frame-bounce  { background: #222438; }
#pin-frame-color  { background: #222438; }
#pin-frame-unpin  { top:100%; }

#examples-2 { z-index: 9999; width: 100%; height: 100%; overflow: hidden; }
#fling-it, #move-it { position: relative; }
#examples-parallax {
	padding-top: 200px;
	height: 800px;
	margin-bottom: -600px;
	overflow: hidden;
}
#parallax-it {
	padding: 0;
	font-size: 80px;
	position: relative;
}

#parallax-it-left {
	position: relative;
	font-size: 120px;
	left: -160px;
	color: #C5C3DE;
	opacity: 0.5;
	-moz-opacity: 0.5;
	filter:alpha(opacity=5);
	overflow: hidden;
}

#parallax-it-right {
	position: relative;
	font-size: 160px;
	left: 80px;
	color: #C5C3DE;
	opacity: 0.25;
	-moz-opacity: 0.25;
	filter:alpha(opacity=2.5);
	overflow: hidden;
	width: 900px;
}

#showcase { padding: 60px 0; }
.gallery { width: 960px; margin: 0 auto; text-align: center; }
.gallery figure { text-align: center; float: left; margin: 10px; height: 270px; display: inline-block; }

#howtouse { padding-top: 200px; }

.divider { font-size: 14px; padding-top: 20px; }

.disclaimer {
	font-size: 18px;
	font-style: italic;
	font-weight: normal;
	margin: 0 auto;
}

#instructions { padding-top: 40px; }

#instructions p { margin: 0px auto 20px; }

#instructions-animation { padding-top: 40px; }

#instructions .documentation {
	font-size: 16px;
	text-align: left;
	width: 720px;
	margin: 10px auto 30px;
	padding: 20px 40px;
	text-shadow: none;
	border: solid 1px #69697A;
	color:#C5C3DE;
}

#instructions .param-list {
	font-size: 18px;
	line-height: 1.4;
	padding: 0 18px;
	margin: 1em 0 0;
}

#instructions .param-list-sub {
	padding-left: 36px;
	font-size: 15px;
}

#instructions .param-list li {
	font-weight: normal;
	padding-bottom: 20px;
	text-shadow: 0 2px 1px #000;
}
.param {
	font-style:italic;
}

.param-list small {
	display: block;
	font-weight: normal;
	font-style: italic;
}

#credits { 
	/*max-width: 860px;
	margin: 0 auto;
	padding: 80px 0; */
}

/* mobile demo */

.mobile-demo #fade-it {
	margin: 400px 0 500px 0;
}
.mobile-demo #examples-1 {
	margin-bottom: 200px;
}
.mobile-demo #instructions {
	width: 800px;
	text-align: left;
	margin: auto;
}
.mobile-demo ul {
	list-style: disc;
	color:#FFFFFF;
	margin: 20px 80px 100px 80px;
}
.mobile-demo #content-wrapper {
	margin-bottom: 100px;
	width: 100%;
}
.mobile-demo h2 {
	width: 90%;
}
.mobile-demo h3 {
	color:#C5C3DE;
}

#newversion {
	position: absolute;
	top: 230px;
	left: 50%;
	margin-left: 250px;
	z-index: 100;
	background-color: #D92B00;
	width: 200px;
	height: 200px;
	border-radius: 100px;
	box-shadow: 0 2px 1px #000;
}
#newversion h2 {
	font-size: 43px;
	margin: 0;
	padding: 30px 0px 0px 0px;
	color: #FFFFFF;
	font-style: normal;
}
#newversion a {
	text-decoration: none;
	padding: 0px 20px 20px 20px;
	display: block;
	font-size: 18px;
	color: #FFFFFF;
}
#newversion a:hover span {
	text-decoration: underline;
}

**Comment by janpaepke** [comment]
Support for Superscrollorama has been suspended.
But working with ScrollMagic shouldn't be a big deal if you come from Superscrollorama.
So feel free to ask about any problems you might have in the [ScrollMagic issues section](https://github.com/janpaepke/ScrollMagic/issues).
